docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+volumes:
+  database2: {}
+
+services:
+  # Named "database2" because "database" conflicts with directory inside project
+  database2:
+    image: mysql:latest
```

Comment:
I would pin the version to ``8`` to not accidentally update to 9 or something.

Reply:
Good point, will update that!